web/tools/webpack.config.babel.js

```diff
@@ -35,6 +35,43 @@ const readVersionName = () => {
   return versionFile.versionName
 }
 
+const generateManifest = (content: string, buildConfigName: string): string => {
```

Comment:
Maybe extract this to a separate script file, what do you think?

Reply:
Also thought about it, but now i removed the switch-case and therefore its a lot shorter. Should i move it nevertheless?

Reply:
No I guess it is fine then.